codalab/worker/local_run/local_run_manager.py

```diff
@@ -105,7 +105,7 @@ def load_state(self):
                     run_state = run_state._replace(
                         container=self._docker.containers.get(run_state.container_id)
                     )
-                except docker.errors.NotFound as ex:
+                except docker.errors.ImageNotFound as ex:
```

Comment:
Again, double check that this is a `Container` `get`, not `Image`

Reply:
Fixed.